src/pages/HttpBin/index.js

```diff
@@ -45,28 +45,27 @@ const HttpBin = () => {
   const [responseIsLoad, setResponseIsLoad] = useState(true);
```

Comment:
the default value should be `false`, right?

Reply:
The same as https://github.com/iviaks/english-games/pull/7#discussion_r488421528